Makefile.PL

```diff
@@ -0,0 +1,105 @@
+use strict;
+use warnings;
+use ExtUtils::MakeMaker;
+
+my $developer = -f '.gitignore';
+ExtUtils::MakeMaker->VERSION(6.98) if $developer;
+
+my %WriteMakefileArgs = (
+    NAME                => 'Text::Glob',
+    VERSION_FROM        => 'lib/Text/Glob.pm',
+    ABSTRACT_FROM       => 'lib/Text/Glob.pm',
+    AUTHOR              => 'Richard Clamp <richardc@unixbeard.net>',
+    LICENSE             => 'perl_5',
+
+    META_MERGE => {
+        'meta-spec' => { version => 2 },
+        dynamic_config => 0,
+        resources => {
+            repository => {
+                url => 'https://github.com/richardc/perl-text-glob.git',
+                web => 'https://github.com/richardc/perl-text-glob',
+                type => 'git',
```

Comment:
It's not required, but it's awfully helpful to be able to find the repository for submitting patches -- metacpan.org provides a link using this data.